test/e2e/node_drain_test.go

```diff
@@ -291,15 +295,19 @@ func generateStatefulset(input generateStatefulsetInput) *appsv1.StatefulSet {
 		Spec: appsv1.StatefulSetSpec{
 			Selector: &metav1.LabelSelector{
 				MatchLabels: map[string]string{
-					"app":         "nonstop",
-					"statefulset": input.Name,
+					"app":               "nonstop",
+					"e2e-test":          "node-drain",
+					boskosResourceLabel: os.Getenv("BOSKOS_RESOURCE_NAME"),
+					"statefulset":       input.Name,
 				},
 			},
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: map[string]string{
-						"app":         "nonstop",
-						"statefulset": input.Name,
+						"app":               "nonstop",
+						"e2e-test":          "node-drain",
+						boskosResourceLabel: os.Getenv("BOSKOS_RESOURCE_NAME"),
```

Comment:
I assume this label is somehow propagated to the PVC? Let's please add godoc comment that it is propagated + that we use/need it for cleanup